frontend/src/lib/components/TaxonomicFilter/types.ts

```diff
@@ -30,6 +30,7 @@ export interface TaxonomicFilterGroup {
     logic?: LogicWrapper
     value?: string
     searchAlias?: string
+    valuesEndpoint?: (key: string) => string
```

Comment:
Q: Do we need the `valuesEndpoint` when `endpoint` does the same job (I think) ? 

Like for Pageview URLs: https://github.com/PostHog/posthog/blob/master/frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.ts#L137